libfwupdplugin/fu-ihex.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2025 Richard Hughes <richard@hughsie.com>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#[derive(ToString)]
+enum FuIhexFirmwareRecordType {
+    Data            = 0x00,
+    Eof             = 0x01,
+    ExtendedSegment = 0x02,
+    StartSegment    = 0x03,
+    ExtendedLinear  = 0x04,
+    StartLinear     = 0x05,
```

Comment:
at least for 0->5, no need to add explicit values, right?

Reply:
Good catch, fixed, thanks.